exporter/datadogexporter/go.mod

```diff
@@ -4,8 +4,8 @@ go 1.20
 
 require (
 	github.com/DataDog/agent-payload/v5 v5.0.89
-	github.com/DataDog/datadog-agent/pkg/proto v0.48.0
-	github.com/DataDog/datadog-agent/pkg/trace v0.48.0
+	github.com/DataDog/datadog-agent/pkg/proto v0.49.0-rc.2
+	github.com/DataDog/datadog-agent/pkg/trace v0.50.0-devel.0.20231030140030-aa8a206e6ac7
```

Comment:
Can you also update the module to this pseudoversion in the other Datadog modules (connector, processor, `internal/datadog`)?

Reply:
Done.